db/db/interface.go

```diff
@@ -0,0 +1,55 @@
+package db
+
+import (
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+)
+
+type Job struct {
+	gorm.Model
+	Title        string
+	Metadata     string
```

Comment:
What does metadata include? Tags? or something else?

Reply:
Metadata will include things like salary and contract type.  This is what we get from indeed, just a string.

Reply:
This is not what we call Tags... that will be our own things we create from processing the description ect.